gpustat.py

```diff
@@ -37,9 +41,11 @@ class ANSIColors:
     BOLD_GREEN  = '\033[1;32m'
     BOLD_YELLOW = '\033[1;33m'
 
-    @staticmethod
-    def wrap(color, msg):
-        return (color + msg + ANSIColors.RESET)
+NOT_SUPPPORTED = 'Not Supported'
+
+@staticmethod
```

Comment:
The wrap is moved from class `ANSIColors` to outside incidentally, which doesn't cause any bugs in my test case. Frustrating~